### PR TITLE
test: dump the namespace content when e2e test cases failed

### PR DESCRIPTION
**Pull request by tokers**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues

___
### Bugfix
- Description

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

___
### Backport patches
- Why need to backport?

- Source branch

- Related commits and pull requests

- Target branch

### CODE REVIEW

File: .github/workflows/e2e-test-ci.yml [hunk 1]
```diff
@@ -73,7 +73,7 @@ jobs:
           submodules: recursive
       - name: Install kind
         run: |
-          curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.10.0/kind-linux-amd64
+          curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.11.0/kind-linux-amd64
```
Comment (tao12345666333) [inline .github/workflows/e2e-test-ci.yml:76]: should we update to v0.11.1 ?

File: .github/workflows/e2e-test-ci.yml [hunk 2]
```diff
@@ -84,6 +84,10 @@ jobs:
         run: |
           go get -u github.com/onsi/ginkgo/ginkgo
           sudo cp ~/go/bin/ginkgo /usr/local/bin
+      - name: Create K8s cluster
+        run: |
+          make kind-up
+          kubectl get nodes
```
Comment (tao12345666333) [inline .github/workflows/e2e-test-ci.yml:90]: should we need add `kubectl wait --for=condition=Ready nodes --all` to wait node ready ?

Comment (tokers) [reply .github/workflows/e2e-test-ci.yml]: Good point! Changed.

File: .github/workflows/e2e-test-ci.yml (hunk 1 above)
Comment (tokers) [reply .github/workflows/e2e-test-ci.yml]: Updated.

**Review by gxthrj — APPROVED** [review]

**Review by tao12345666333 — APPROVED** [review]

**Pull request merged**